core/trino-main/src/main/java/io/trino/sql/planner/plan/AggregationNode.java

```diff
@@ -119,6 +119,16 @@ public GroupingSetDescriptor getGroupingSets()
         return groupingSets;
     }
 
+    /**
+     * Returns {@code true} if the aggregation has form GROUP BY ().
```

Comment:
This should be documented using `@return` in the javadocs:

> `@return` true if the aggregation collapses all rows into a single global group (e.g., as a result of a `GROUP BY ()` query). Otherwise, false.